client/src/Components/GameOver/GameOver.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro';
+import Button from '../Button/Button';
+import Header from '../Header/Header';
+import { useSetPlayerLives } from '../../Services/Context';
+import { useHistory } from 'react-router-dom';
+
+export default function GameOver() {
+  const history = useHistory();
+  const setPlayerLives = useSetPlayerLives();
+  function handleClick() {
+    history.push('/');
+  }
+  return (
+    <GameOverWrapper>
+      <Header>Game Over!</Header>
+      <Button
```

Comment:
You can use a self closing tag here as you don't have any children but only props 
```suggestion
        <Button ... />
```